test/test_lint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I wrote this horrible Bash script a few years ago to have a quick test for our new CI pipeline. It wasn't meant to stick around. And now it keeps on coming back to haunt me 🙈😅

1. We don't need the script. We are just executing a single command. Why wrap it? It's just complicating things and makes it more difficult to read/understand what's happening. Just put the command directly in the CI workflow. GitHub Actions will clean after itself, so no issues with that. And all the other stuff is just boilerplate to support the wrapper script, not the test itself.
2. Even if a script _were_ useful here (e.g., because we would need to run multiple complex commands that would just blow up and make the CI workflow difficult to read), we should write a clean Python script that we could then run with a test runner like `pytest`. And that is written specifically for the use case (this script was written for a use case where multiple commands were executed and where an explicit, manual clean up was necessary).